test/Makefile

```diff
@@ -97,14 +102,14 @@ test: $(TESTS)
 	done
 
 clean:
-	$(RM) $(TESTS) $(GTEST_LIB) $(TEST_OBJS) $(UTIL_OBJS)
+	$(RM) $(TESTS) $(GTEST_LIB) $(GMOCK_LIB) $(TEST_OBJS) $(UTIL_OBJS)
 
 purge: clean
 	$(RM) init-submodules
-	(cd .. && git submodule deinit -f $(GTEST_MODULE:../%=%))
+	(cd .. && git submodule deinit -f $(GTEST_MODULE:../%=%) && git submodule deinit -f $(GMOCK_MODULE:../%=%))
```

Comment:
there is no `GMOCK_MODULE` defined yet.

Reply:
if we end up `deinit` 2 paths, you can use single command:
```
git submodule deinit -f $(GTEST_MODULE:../%=%) ${GMOCK_MODULE:../%=%)}
```

Reply:
No longer needed.